docs/versioned_docs/version-2.50.0-LTS/widgets/timeline.md

```diff
@@ -34,58 +35,57 @@ The **Timeline** component can be used to do a visual representation of a sequen
+There are currently no exposed variables for the Timeline component.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## General
 ### Tooltip
 
-A Tooltip is often used to specify extra information when the user hovers the mouse pointer over the component.
-
-Under the <b>General</b> accordion, you can set the value in the string format. Now hovering over the component will display the string as the tooltip.
+A Tooltip provides additional information when users hover over the component. Set the tooltip content under the **General** accordion in the component's properties.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
-|:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Property        | Description                               | Expected Value |
+| :-------------- | :---------------------------------------- | :------------- |
+| Show on desktop | Controls the component's desktop visibility | Set to `{{true}}` to show on desktop, `{{false}}` to hide |
+| Show on mobile  | Controls the component's mobile visibility  | Set to `{{true}}` to show on mobile, `{{false}}` to hide |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Styles
 
-| <div style={{ width:"100px"}}> Style </div>     | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"100px"}}> How To Access </div> |
-|:----------- |:----------- |:--------|
-| Visibility | Toggle on or off to control the visibility of the component. | Programmatically change its value by clicking on the `Fx` button next to it. If `{{false}}` the component will not visible after the app is deployed. By default, it's set to `{{true}}`. |
+| Property | Description | Configuration Options |
+| :------- | :---------- | :-------------------- |
+| Visibility | Controls the component's visibility | Toggle on/off or use `{{true}}` / `{{false}}` |
+| Box Shadow | Adds a shadow effect around the component | Options: `none` (default), `sm`, `md`, `lg`, `xl`, `2xl` |
 
 :::info
-Any property having `Fx` button next to its field can be **programmatically configured**.
+Any property with an `Fx` button next to it can be **programmatically configured**.
```

Comment:
The fx should be bold and not enclosed in single ticks. Please refer to the [style guide](https://docs.tooljet.com/docs/contributing-guide/documentation-guidelines/style-guide/#1-text-formatting-guidelines) for details.